src/next-steps/next-steps.module.css

```diff
@@ -8,18 +8,46 @@
 
 .task {
 	display: flex;
+	padding-right: 10px;
```

Comment:
**Nit (non-blocking):** For all the padding values here, do you mind updating them to be in `rem` instead of `px`? You can leave all the borders etc as pixels, I've just been trying to do all font-sizes and padding/margins in `rem` so things scale with font scaling a bit easier! 😄

Reply:
Done! Updated all padding values to use `rem` instead of `px` in [85c21db](https://github.com/Automattic/bugomattic/pull/71/commits/85c21db2c753083288de429f922cee7293939356)